wled00/bus_manager.cpp

```diff
@@ -402,13 +402,21 @@ BusPwm::BusPwm(BusConfig &bc)
 {
   if (!isPWM(bc.type)) return;
   unsigned numPins = numPWMPins(bc.type);
-#ifdef ESP8266
+  #ifdef ESP8266
   _frequency = bc.frequency ? bc.frequency : WLED_PWM_FREQ;
   // duty cycle resolution (_depth) can be extracted from this formula: CLOCK_FREQUENCY > _frequency * 2^_depth
   for (_depth = MAX_BIT_WIDTH; _depth > 8; _depth--) if (((CLOCK_FREQUENCY/_frequency) >> _depth) > 0) break;  
   analogWriteRange((1<<_depth)-1);
   analogWriteFreq(_frequency);
-#else
+  for (unsigned i = 0; i < numPins; i++) {
+  uint8_t currentPin = bc.pins[i];
+  if (!pinManager.allocatePin(currentPin, true, PinOwner::BusPwm)) {
```

Comment:
pin manager allows for allocation of multiple pins in one go. use `pinManager.allocateMultiplePins()`

Reply:
that is not my code :)